src/awkward/_v2/operations/ak_from_rdataframe.py

```diff
@@ -3,12 +3,12 @@
 import awkward as ak
 
 
-def from_rdataframe(data_frame, column):
+def from_rdataframe(data_frame, columns):
     """
     Args:
         data_frame (`ROOT.RDataFrame`): ROOT RDataFrame to convert into an
              Awkward Array.
-         column (str): A column to be converted to Awkward Array.
+         columns (str or tuple of str): A column or multiple columns to be converted to Awkward Array.
```

Comment:
@jpivarski - now that we've re-introduced `columns`, should we also re-introduce `excluded`?

Reply:
What was `excluded`?

(If we have a way of making a positive statement, the `columns` we want to return, then there shouldn't also be a way of making the negative statement, the columns to be `excluded`. Otherwise, which list wins? But maybe you meant something else by that.)

Reply:
> What was `excluded`?
> 
> (If we have a way of making a positive statement, the `columns` we want to return, then there shouldn't also be a way of making the negative statement, the columns to be `excluded`. Otherwise, which list wins? But maybe you meant something else by that.)

Yes, I agree. It was the list of columns not to include: either take some `columns` or take all but `excluded`.